### PR TITLE
Create .gitignore file

### PR DESCRIPTION
**Pull request by barykine**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/venv
+
+**/__pycache__/
+**/.pytest_cache
+**/*.pyc
+
+# ignore local pyenv settings
+.python-version
```
Comment (Vegter) [inline .gitignore:8]: This should probably be put in a global (private) .gitignore as it is not something that is required for the repo.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+htmlcov
+.coverage
+
+.env
+.idea
```
Comment (Vegter) [inline .gitignore:14]: These are not required for this repo I think? same holds for pytest_cache

**Review by Vegter — COMMENTED** [review]
Some small comments

**Comment by barykine** [comment]
Not needed as we are not running Airflow locally (yet), only in Docker.

**Pull request closed without merging**